### DISCUSSION TITLE
Vimar cover window 14954

### DISCUSSION THREAD

**Discussion by hexstudy** [Koenkk/zigbee2mqtt#18608] [Devices]
Hi,
I have some problem with this Device.
I use a sonoff coordinator with docker installation of z2m

When i use manualy the vimar command my ha don't update but I modify the Device by ha work correctly.
I don't have a correctly state open/closed When che position is different of 0 or 100

The other problem is scene with this type of Device in ha don't modify my cover window.

**Comment by LeoeLeoeL** [comment]
I just migrated the first of 5 14954 from Deconz to Z2M by Sonoff "E".
Same behaviour; Z2M doesn't update the position.
The only way to update is:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/65825223/0e11e0ed-a900-4dee-9fd0-9efdce3a4a28)
